docs/src/pages/_app.page.tsx

```diff
@@ -47,6 +49,12 @@ if (typeof window === 'undefined') {
 function MyApp({ Component, pageProps }) {
   const [expanded, setExpanded] = React.useState(false);
 
+  Storage.addPluggable(new MyStorageProvider('fast', { delay: 10 }));
```

Comment:
I tried adding these lines to`_document.page` but I think because it renders on the server or at build time it won't work. These need to happen in the client. Maybe there is a better place for them?

Reply:
Can we just leave it here?

Reply:
Yea we can leave them here, just didn't know if there was a better place to put them.